nhost/compose/config.go

```diff
@@ -505,23 +473,82 @@ func (c Config) storageService() *types.ServiceConfig {
+		"AUTH_CLIENT_URL":                           authConf.GetRedirections().GetClientUrl(),
+		"AUTH_WEBAUTHN_ENABLED":                     fmt.Sprint(authConf.GetMethod().GetWebauthn().GetEnabled()),
+		"AUTH_WEBAUTHN_RP_NAME":                     authConf.GetMethod().GetWebauthn().GetRelyingParty().GetName(),
+		"AUTH_WEBAUTHN_RP_ORIGINS":                  strings.Join(authConf.GetMethod().GetWebauthn().GetRelyingParty().GetOrigins(), ","),
+		"AUTH_WEBAUTHN_ATTESTATION_TIMEOUT":         fmt.Sprint(authConf.GetMethod().GetWebauthn().GetAttestation().GetTimeout()),
+		"AUTH_ANONYMOUS_USERS_ENABLED":              fmt.Sprint(authConf.GetMethod().GetAnonymous().GetEnabled()),
+		"AUTH_DISABLE_NEW_USERS":                    fmt.Sprint(!authConf.GetSignUp().GetEnabled()),
+		"AUTH_ACCESS_CONTROL_ALLOWED_EMAILS":        strings.Join(authConf.GetUser().GetEmail().GetAllowed(), ","),
+		"AUTH_ACCESS_CONTROL_ALLOWED_EMAIL_DOMAINS": strings.Join(authConf.GetUser().GetEmailDomains().GetAllowed(), ","),
+		"AUTH_ACCESS_CONTROL_BLOCKED_EMAILS":        strings.Join(authConf.GetUser().GetEmail().GetBlocked(), ","),
+		"AUTH_ACCESS_CONTROL_BLOCKED_EMAIL_DOMAINS": strings.Join(authConf.GetUser().GetEmailDomains().GetBlocked(), ","),
+		"AUTH_PASSWORD_MIN_LENGTH":                  fmt.Sprint(authConf.GetMethod().GetEmailPassword().GetPasswordMinLength()),
+		"AUTH_PASSWORD_HIBP_ENABLED":                fmt.Sprint(authConf.GetMethod().GetEmailPassword().GetHibpEnabled()),
+		"AUTH_USER_DEFAULT_ROLE":                    authConf.GetUser().GetRoles().GetDefault(),
+		"AUTH_USER_DEFAULT_ALLOWED_ROLES":           strings.Join(authConf.GetUser().GetRoles().GetAllowed(), ","),
+		"AUTH_LOCALE_DEFAULT":                       authConf.GetUser().GetLocale().GetDefault(),
+		"AUTH_LOCALE_ALLOWED_LOCALES":               strings.Join(authConf.GetUser().GetLocale().GetAllowed(), ","),
+		"AUTH_EMAIL_PASSWORDLESS_ENABLED":           fmt.Sprint(authConf.GetMethod().GetEmailPasswordless().GetEnabled()),
+		"AUTH_SMS_PASSWORDLESS_ENABLED":             fmt.Sprint(authConf.GetMethod().GetSmsPasswordless().GetEnabled()),
+		"AUTH_EMAIL_SIGNIN_EMAIL_VERIFIED_REQUIRED": fmt.Sprint(authConf.GetMethod().GetEmailPassword().GetEmailVerificationRequired()),
+		"AUTH_ACCESS_CONTROL_ALLOWED_REDIRECT_URLS": strings.Join(authConf.GetRedirections().GetAllowedUrls(), ","),
+		"AUTH_MFA_ENABLED":                          fmt.Sprint(authConf.GetTotp().GetEnabled()),
+		"AUTH_MFA_TOTP_ISSUER":                      authConf.GetTotp().GetIssuer(),
+		"AUTH_ACCESS_TOKEN_EXPIRES_IN":              fmt.Sprint(authConf.GetSession().GetAccessToken().GetExpiresIn()),
+		"AUTH_REFRESH_TOKEN_EXPIRES_IN":             fmt.Sprint(authConf.GetSession().GetRefreshToken().GetExpiresIn()),
+		"AUTH_EMAIL_TEMPLATE_FETCH_URL":             "", // TODO: deprecated?
+		"AUTH_JWT_CUSTOM_CLAIMS":                    "", // TODO: ???
+		"AUTH_CONCEAL_ERRORS":                       "", // TODO: ???
```

Comment:
Couldn't find in new config schema

Reply:
@plmercereau ?

Reply:
we haven't put this new config element in the cue schema (and it's not available in the cloud), I would say we should add sth like 
```go
#Auth: {
  concealErrors: bool | *false
}
```
What do you think @elitan @nunopato ?

Reply:
Let's ignore it for now.